PowerEditor/installer/nativeLang/venetan.xml

```diff
@@ -0,0 +1,1255 @@
+o in na carteƚa che necesita de i priviƚegi de scritura per poterghe asedare.
+Le to inpostasioni ne el cloud ƚe vegnerà scancelà. Reinposta on vaƚore coerente da ƚe inpostasioni."/>
+            <FilePathNotFoundWarning title="Vertaura file" message="El file che se stà sercando de vèrxare no&apos;l existe."/>
+            <SessionFileInvalidError title="Inposibiƚe caricare ƚa sesion" message="El file de sesion el xe coroto o no&apos;l xe vaƚido."/>
+            <DroppingFolderAsProjectModeWarning title="Asion no vaƚida" message="Te poi portare file o carteƚe, ma no entranbe, parché te si in modaƚità riƚasia Carteƚa come Progetto.
+te devi abiƚitare &quot;Vèrxi tuti i file de ƚa carteƚa invece de aviare Carteƚa come Workspace al trasinamento&quot; ne ƚa sesion &quot;Carteƚa predefinìa&quot; de ƚe prefarense per ativare sta funsionaƚità."/>
+            <SortingError title="Erore ordinamento" message="Inposibiƚe efetuare on ordinamento numerico a causa de ƚa riga $INT_REPLACE$."/>
+            <ColumnModeTip title="Consijio modaƚità coƚona" message="Utiƚixa &quot;ALT+Seƚesion Mouse&quot; o &quot;Alt+Shift+Tasti frece&quot; par pasare a ƚa modaƚità coƚona."/>
+            <BufferInvalidWarning title="Salvatagio falìo" message="Inposibiƚe salvare: el buffer no&apos;l xe vàƚido."/>
+            <DoCloseOrNot title="Mantien file no existente" message="El file &quot;$STR_REPLACE$&quot; no&apos;l existe pì.
+Mantenere el file vèrto?"/>
+            <DoDeleteOrNot title="Eƚimina file" message="El file &quot;$STR_REPLACE$&quot;
+vegnerà spostà nel Cestino e sto documento el vegnerà sarà.
+Continuare?"/>
+            <NoBackupDoSaveFile title="Salva" message="El to backup no&apos;l poƚe èsare trovà (scanceƚà da altrove).
+Salvaƚo o i to dati i vegnerà persi.
+Vuto salvare el file &quot;$STR_REPLACE$&quot; ?"/>
+            <DoReloadOrNot title="Ricarica" message="&quot;$STR_REPLACE$&quot;
+Sto file el xe xà stà modificà da &apos;n altro programa.
+Vuto ricaricarƚo?"/>
+            <DoReloadOrNotAndLooseChange title="Ricarica" message="&quot;$STR_REPLACE$&quot;
+Sto file el xe stà modificà da &apos;n altro programa.
+Vuto ricaricarƚo e perdare tute ƚe modifiche aportà in Notepad++?"/>
+            <PrehistoricSystemDetected title="Sistema preistorico trovà" message="Senbra che te si drìo uxare on sistema preistorico. Sta funsionaƚità ƚa necesita de un sistema pì moderno, scusane."/>
+            <XpUpdaterProblem title="Axornamento Notepad++" message="Ƚ&apos;axornamento automatico de Notepad++ no&apos;l xe compatibiƚe con XP a causa de un sistema de permesi obsoƚeto.
+Vuto &apos;ndare ne ƚa pàgina de Notepad++ par scaricare l&apos;axornamento manualmente?"/>
+            <DocTooDirtyToMonitor title="Problema monitoragio" message="El documento el xe sporco. Salva ƚe modifiche prima de aviare el monitoragio."/>
+            <DocNoExistToMonitor title="Probƚema monitoragio" message="El 
```

Comment:
<find-in-files-filter-tip value="Ruma en cpp, cxx, h, hxx &amp;&amp; hpp:&#xD;*.cpp *.cxx *.h *.hxx *.hpp&#xD;&#xD;Ruma en tuti i file a ecesion de exe, obj &amp;&amp; log:*.* !*.exe !*.obj !*.log"/>